crates/db_views/src/comment_view.rs

```diff
@@ -376,22 +376,24 @@ impl<'a> CommentQueryBuilder<'a> {
       query = query.filter(comment::content.ilike(fuzzy_search(&search_term)));
     };
 
-    query = match self.listing_type {
-      // ListingType::Subscribed => query.filter(community_follower::subscribed.eq(true)),
-      ListingType::Subscribed => query.filter(community_follower::person_id.is_not_null()), // TODO could be this: and(community_follower::person_id.eq(person_id_join)),
-      ListingType::Local => query.filter(community::local.eq(true)),
-      _ => query,
-    };
+    if let Some(listing_type) = self.listing_type {
+      query = match listing_type {
+        // ListingType::Subscribed => query.filter(community_follower::subscribed.eq(true)),
```

Comment:
Whats with this commented line?

Reply:
Was an alternate way to do that, removed.